libraries/ms-common/src/ms_test_helper_can.c

```diff
@@ -0,0 +1,25 @@
+#include "ms_test_helper_can.h"
+#include "can.h"
+#include "can_msg_defs.h"
+#include "event_queue.h"
+#include "gpio.h"
+#include "interrupt.h"
+#include "soft_timer.h"
+
+StatusCode initialize_can_in_test(CanStorage *storage, SystemCanDevice device, EventId tx_event,
+                                  EventId rx_event, EventId fault_event) {
+  event_queue_init();
+  gpio_init();
+  interrupt_init();
+  soft_timer_init();
```

Comment:
Might be unclear that this is happening if someone else were to use this.

 Also calling `_init` more than once should result in an error ... `status_ok_or_return` at the very least?

Reply:
maybe change to something like `initialize_can_and_dependencies_in_test` or similar given that the convention is to just include the inits in the setup_test function